### DISCUSSION TITLE
WSL folder detection

### DISCUSSION THREAD

**Discussion by goatedpenguin** [microsoft/WSL#10929] [Q&A]
So I got a new laptop with windows on it and I have a wsl installation with all my data for that distro on my pc. Is there a way I could move the folder from the pc to the laptop and WSL can detect the files etc. automatically on the laptop?
Thanks in advance. :)

**Comment by zcobol** [comment]
It's doable! Copy the `ext4.vhdx` from your pc to the laptop and use it to setup a new distro.

See import option:

```
    --import <Distro> <InstallLocation> <FileName> [Options]
        Imports the specified tar file as a new distribution.
        The filename can be - for stdin.

        Options:
            --version <Version>
                Specifies the version to use for the new distribution.

            --vhd
                Specifies that the provided file is a .vhdx file, not a tar file.
                This operation makes a copy of the .vhdx file at the specified install location. 
```

**Reply by goatedpenguin** [reply]
So all I have to do is the export command which will make it tar send it to the laptop and then I can just import it?
